lib/hooks/typescript-compilation.ts

```diff
@@ -4,15 +4,15 @@ import fiberBootstrap = require("./../common/fiber-bootstrap");
 import { TARGET_FRAMEWORK_IDENTIFIERS } from "../common/constants";
 
 fiberBootstrap.run(() => {
-	$injector.require("typeScriptCompilationService", "./common/services/typescript-compilation-service");
 	let project: Project.IProject = $injector.resolve("project");
 	if (!project.projectData) {
 		return;
 	}
 
-	let typeScriptFiles = project.getTypeScriptFiles().wait();
+	let typeScriptService: ITypeScriptService = $injector.resolve("typeScriptService");
+	let typeScriptFiles = typeScriptService.getTypeScriptFiles(project.getProjectDir().wait()).wait();
```

Comment:
You can use `project.projectDir` instead of `project.getProjectDir().wait()`, not a merge-stopper though